packages/libp2p/test/connection-manager/index.spec.ts

```diff
@@ -208,7 +208,7 @@ describe('Connection Manager', () => {
         connectionManager: {
           maxConnections: max,
           allow: [
-            '/ip4/83.13.55.32'
+            '/ip4/83.13.55.32/ipcidr/32'
```

Comment:
If `/ip4/83.13.55.32` won't work, we need to release this as a breaking change. I think we should still accept specific IP addresses

Reply:
@acul71 let me know if this is not clear enough :) we probably just need to check if the supplied `allow`/`deny` is an `ipcidr` multiaddr or not, and if not, append `/ipcidr/32` internally to create the IpNet

Reply:
@SgtPooki I was tempted to to the first commented way then I choose to go with the second.
PR soon. Ciao

```typescript
// allow/deny lists
    /**
    this.allow = (init.allow ?? []).map(a => {
      if (a.includes('/ipcidr/')) {
        return convertToIpNet(multiaddr(a))
      } else {
        return convertToIpNet(multiaddr(a + '/ipcidr/32'))
      }
    })
    this.deny = (init.deny ?? []).map((a) => {
      if (a.includes('/ipcidr/')) {
        return convertToIpNet(multiaddr(a))
      } else {
        return convertToIpNet(multiaddr(a + '/ipcidr/32'))
      }
    })
    */
    // allow/deny lists
    this.allow = (init.allow ?? []).map((a) => {
      try {
        // Try to parse a as a multiaddr directly
        return convertToIpNet(multiaddr(a))
      } catch (error) {
        // If parsing fails, append '/ipcidr/32' and try again
        return convertToIpNet(multiaddr(a + '/ipcidr/32'))
      }
    })

    this.deny = (init.deny ?? []).map((a) => {
      try {
        // Try to parse a as a multiaddr directly
        return convertToIpNet(multiaddr(a))
      } catch (error) {
        // If parsing fails, append '/ipcidr/32' and try again
        return convertToIpNet(multiaddr(a + '/ipcidr/32'))
      }
    })
```